Gemfile

```diff
@@ -1,3 +1,5 @@
 source 'https://rubygems.org'
 gem 'github-pages', group: :jekyll_plugins
 
+
+gem "webrick", "~> 1.7"
```

Comment:
I _think_ we should revert this change, but I'm curious why it was added.

If it was for testing/previewing the page locally, we can already do this with Bundler: https://tableau.github.io/server-client-python/docs/dev-guide#running-docs-locally

Reply:
@bcantoni I inadvertently added this - is there an easy way to remove this file?

Reply:
yep, perfect!